src/main/resources/assets/colytra/lang/fr_fr.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tooltip.colytra.broken": "Inutilisable",
+  "text.autoconfig.colytra.title": "Colytra",
+  "text.autoconfig.colytra.option.permissionMode": "Mode de permission",
+  "text.autoconfig.colytra.option.permissionMode.@Tooltip[0]": "Défini si la liste de permission est une blacklist ou une whitelist",
+  "text.autoconfig.colytra.option.permissionMode.@Tooltip[1]": "BLACKLIST : Les objets spécifiés ne peuvent être combiné avec des élytres",
+  "text.autoconfig.colytra.option.permissionMode.@Tooltip[2]": "WHITELIST : Seulement les objets spécifiés peuvent être combinés avec des élytres",
+  "text.autoconfig.colytra.option.permissionList": "Liste de permission",
+  "text.autoconfig.colytra.option.permissionList.@Tooltip": "Liste des objets par nom de registre à blacklister/whitelister selon le mode de permission",
+  "text.autoconfig.colytra.option.colytraMode": "Mode de Colytra",
+  "text.autoconfig.colytra.option.colytraMode.@Tooltip[0]": "Défini le comportement des plastrons élytres",
+  "text.autoconfig.colytra.option.colytraMode.@Tooltip[1]": "NORMAL : Les élytres sont indépendantes du plastron et peuvent être séparées plus tard",
+  "text.autoconfig.colytra.option.colytraMode.@Tooltip[2]": "UNISON : Les élytres fusionneront avec le plastron et ne pourront plus être séparées",
```

Comment:
**L'élytre fusionne sa durabilité** avec le plastron et ne **peut** plus être séparé**e**

Reply:
The original english text does not refer to durability.

Reply:
Should the translation be better than the original, good question :)